src/components/MainPage.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import { Header } from 'semantic-ui-react';
+import Articles from '../modules/Articles';
+import ArticleCard from './ArticleCard';
+import BreakingNews from './layout/BreakingNews';
+
+const MainPage = () => {
+  const [articles, setArticles] = useState([]);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const fetchArticles = async () => {
+    try {
+      const response = await Articles.get()
+      setArticles(response);
+      setErrorMessage('');
+    } catch (error) {
+      if (error.response.status === 500) {
+        setErrorMessage(
+          'Servers are currently not responding, Pleas try again later'
+        );
+      } else {
+        setErrorMessage(error.message);
+      }
+    }
+  };
```

Comment:
If your module looks something like this, then all you need in the `useEffect` is Articles.get(functions)

```
async get(setArticles, setErrorMessage) {
  try {
    let response = await axios.get('/articles')
    setArticles(response.data.articles);
    setErrorMessage('');
  } catch (error) {
    // ...
  }
}
```

Reply:
fixed

Reply:
fixed